hackathon2.R

```diff
@@ -0,0 +1,90 @@
+# GOAL of this research: ARE WE IMPROVING on SCREENING of MUAC and OEDEMA? 
+##Are wee expanding our screening coverage? From baseline vs. endline? By sex? By location?
+
+## Install & Load R packages 
+install.packages("gtsummary")
+install.packages("dplyr")
+
+library(dplyr)
+library("gtsummary")
+
+## Load functions in R directory 
+for (f in list.files("R", full.names = TRUE)) source (f)
+
+# Read data 
+nut<-read.csv(file="https://raw.githubusercontent.com/OxfordIHTM/ihtm-hackathon-2023/main/data/nutrition_survey_data.csv")  
+nut
+
+##Subset data: Transform variables of interest into table nut1 
+nut1 <- nut %>% select(survey_round,county,sex,muac_screen,oedema_screen)
+
+View(nut1)
+
+##Clean data in subset: Change 99 to NA 
+###Clean data only after having made subset nut1. Code don't work if data is cleaned in nut and then subset. 
+nut1$muac_screen <- ifelse(nut1$muac_screen == 99, "NA", nut1$muac_screen)
+nut1$muac_screen [nut$muac_screen == 1] <- "Y"
+nut1$muac_screen [nut$muac_screen == 2] <- "N"
+
+nut1$oedema_screen <- ifelse(nut1$oedema_screen == 99, "NA", nut1$muac_screen)
+nut1$oedema_screen [nut$oedema_screen == 1] <- "Y"
+nut1$oedema_screen [nut$oedema_screen == 2] <- "N"
+
+nut1$sex <- ifelse(nut1$sex == 1, "Male", nut1$sex)
+nut1$sex <- ifelse(nut1$sex == 2, "Female", nut1$sex)
+
+View(nut1)
+
+##1. What is the coverage of mid-upper arm circumference (MUAC) screening among the children in the sample?
+## Use tbl_summary to compare muac_screen by survey round, endline vs. baseline
+nut1 %>% tbl_summary(by = survey_round, 
+                      include=c(muac_screen),
+                      statistic = list(all_continuous() ~ "{mean} ({sd})"))
+                                     
+nut1
+
+### The endline data shows LITTLE if NO improvement in MUAC screening coverage among the sample children vs. baseline. Percentage of children had MUAC screened (answered 1 to survey) stayed relatively the same from baseline to endline. The percentage of children without MUAC screening went up from 73% to 93%, mainly due to children from the NA population at baseline moving over. This may suggest a better awareness in this population related to not having gotten MUAC screening?
```

Comment:
Just a note here. Remember that the baseline and endline sample are independent of each other. They are not the same children. So, any shifts in those refusing to answer the question (NA value) for MUAC screening or oedema screening from baseline to endline is inconsequential to the actual results. They just need to be removed from the denominator of the analysis for prevalence/coverage.